contracts/option_pricing.cairo

```diff
@@ -13,12 +13,25 @@
 //
 
 from starkware.cairo.common.cairo_builtins import HashBuiltin
-from starkware.cairo.common.math import sign, assert_le, unsigned_div_rem
+from starkware.cairo.common.math import sign, assert_le, unsigned_div_rem, abs_value
 from starkware.cairo.common.math_cmp import is_le
+from starkware.cairo.common.bool import TRUE, FALSE
 
 // Third party imports. Was copy pasted to this repo.
 from math64x61 import Math64x61
 
+from contracts.types import Bool
+
+func max{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
```

Comment:
max is also implemented in helpers.cairo. Is there a reason why not use that?

Reply:
It's due to circular imports, since amm.cairo imports both from helpers and option pricing, option pricing can't import from helpers

Reply:
Helpers don't import AMM, though, so not circular (Or is this a problem in Cairo?)